mm2src/coins/lp_coins.rs

```diff
@@ -1570,6 +1593,24 @@ pub async fn lp_coininit(ctx: &MmArc, ticker: &str, req: &Json) -> Result<MmCoin
             let token = SlpToken::new(*decimals, ticker.into(), token_id.clone().into(), platform_coin, confs);
             token.into()
         },
+        // Only BTC is supported for now
+        CoinProtocol::LIGHTNING { platform, network } => {
```

Comment:
I think it's time to stop adding new protocols to the old `lp_coininit`. Could you please add `l2.rs` (layer 2) and `lightning_activation.rs` to the `coins_activation` crate similarly to `token.rs` and `slp_token_activation.rs`?

Reply:
I added `l2.rs` and `lightning_activation.rs`. Not sure if I should remove lightning from `lp_coininit` or not. what do you think?

Reply:
> Not sure if I should remove lightning from lp_coininit or not. what do you think?

Please feel free to remove it from `lp_coininit`. I think it doesn't worth the effort to keep adding new protocols there.